### PR TITLE
Add func: allow auto snapshot during expanding volume.

### PR DESCRIPTION
**Pull request by AlbeeSo**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
Add func : allows to use auto snapshot during expanding volume of ESSD.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/disk/controllerserver.go [hunk 1]
```diff
@@ -103,6 +103,20 @@ const (
 	MinimumDiskSizeInGB = 20
 	// MinimumDiskSizeInBytes ...
 	MinimumDiskSizeInBytes = 21474836480
+	//volumeExpandAutoSnapshotIDKey is the key of volumeExpandAutoSnapshotID on pvc
+	volumeExpandAutoSnapshotIDKey = "volumeExpandAutoSnapshotID"
+	// volumeExpandAutoSnapshotNameKey is the key of volumeExpandAutoSnapshotIName on pvc
+	volumeExpandAutoSnapshotNameKey = "volumeExpandAutoSnapshotName"
```
Comment (mowangdk) [inline pkg/disk/controllerserver.go:109]: IMO, Snapshot name is not necessary to mark on pvc?

Comment (AlbeeSo) [reply pkg/disk/controllerserver.go:109]: controllerServer manages a map to record the created snapshot,  with the names of snapshot as the keys.

Comment (AlbeeSo) [reply pkg/disk/controllerserver.go:109]: Sure it's okay to only mark the ID if the volumeExpandAutoSnapshots are not recorded in the controllerServer then the nodeServer will not need to delete it anymore.

**Comment by mowangdk** [comment]
@AlbeeSo  Run ./hack/update-gofmt.sh  & repush to fix fmt err

**Pull request merged**